data/org.freedesktop.impl.portal.FileChooser.xml

```diff
@@ -66,6 +66,13 @@
               See org.freedesktop.portal.FileChooser.OpenFile() for details.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>current_filter (sa(us))</term>
+            <listitem><para>
+              Filter to set at dialog creation.
+              If the filters list is nonempty, it should be present there.
```

Comment:
This bit of documentation is somewhat unclear (admittedly already the case in gtk): Does this mean it is an error if there is not an identical variant in the filter list, or that the caller can expect this filter to be added to the list that is displayed in the UI ?

Reply:
For GTK+, it's the former. The filter will not be added to the list displayed in the UI.

For Qt... the documentation is not at all clear. I guess we can document the GTK+ behavior, and if Qt wants to do something different it can just add a new option to the dict.